docs/intro.rst

```diff
@@ -0,0 +1,142 @@
+Introduction to distgen
+=======================
+
+`Distgen <https://github.com/devexp-db/distgen/>`_ is a distribution-oriented
+templating system.
+
+The problem this project tries to mitigate is "portable" scripting for variety
+of operating systems (currently Linux distributions only) in the wild.  While
+writing a "universal" script, one needs to take into account small or bigger
+differences among operating systems (like package installation tools, versions
+of utilities, expected directories for binaries, libraries, etc.).
+
+The *distgen* project is thus something like database of OS differences together
+with convenience tool-set that allows you to instantiate valid script for
+particular distribution.  The concept is to have *template* file (mostly raw
+jinja2 template) together with "declarative" *spec* file (YAML file) that
+fulfils the needs of particular template.
+
+
+Download/Installation
+---------------------
+
+Stable releases of distgen are available (for Red Hat distributions) in
+`Copr <https://copr.fedoraproject.org/coprs/praiskup/distgen>`_.
+
+You can also run development version directly from
+`Github <https://github.com/devexp-db/distgen/>`_, simply use the
+``dg`` shell wrapper available in git root directory.
+
```

Comment:
...which fails with a traceback if you don't have the yaml module installed. Maybe worth mentioning...? If not here, in the README at least perhaps

Reply:
Right, I should mention installation of dependencies - thanks!